### PR TITLE
Change help for makegroupchat

### PR DESCRIPTION
**Pull request by sirKiraUzumaki**

### CODE REVIEW

File: server/chat-commands/room-settings.ts [hunk 1]
```diff
@@ -784,6 +784,7 @@ export const commands: ChatCommands = {
 	makegroupchathelp: [
 		`/makegroupchat [roomname] - Creates an invite-only group chat named [roomname].`,
 		`/subroomgroupchat [roomname] - Creates a subroom groupchat of the current room. Can only be used in a public room you have staff in.`,
+		`Requires: % @ # &`,
```
Comment (xInstruct) [inline server/chat-commands/room-settings.ts:787]: I mean, global voices can make groupchats too.

Might want to modify it to include that I am but not sure how to do it succinctly.

Comment (sirKiraUzumaki) [reply server/chat-commands/room-settings.ts:787]: maybe it can be like this: Requires: Trusted Users

Comment (sirKiraUzumaki) [reply server/chat-commands/room-settings.ts:787]: maybe I should add it in line 785

Comment (AnnikaCodes) [inline server/chat-commands/room-settings.ts:787]:
```suggestion
		`Only users who are staff in a public room or global auth can make groupchats.`,
```
This is how I would word it.

**Comment by AnnikaCodes** [comment]
Also, as per [our commit message guidelines](https://github.com/smogon/pokemon-showdown/blob/master/CONTRIBUTING.md#cm3-grammar), your PR name shouldn't end in a period.

**Comment by sirKiraUzumaki** [comment]
> Also, as per [our commit message guidelines](https://github.com/smogon/pokemon-showdown/blob/master/CONTRIBUTING.md#cm3-grammar), your PR name shouldn't end in a period.

Opps sorry, I will make sure not to do this again.

**Review by AnnikaCodes — APPROVED** [review]
Looks fine to me. Thanks!

**Pull request merged**